frontend/app/routes/addMaker.tsx

```
@@ -154,14 +158,33 @@ export default function AddMaker() {
+                    onChange={handleChange}
+                    placeholder="Optional"
+                    className="w-full px-4 py-2.5 pr-10 bg-gray-800 border border-gray-700 rounded-lg focus:border-orange-500 focus:outline-none text-gray-100 placeholder-gray-500"
+                  />
+                  <button
+                    type="button"
+                    onClick={() => setShowPassword((p) => ({ ...p, password: !p.password }))}
+                    className="absolute inset-y-0 right-0 px-3 flex items-center text-gray-400 hover:text-gray-200"
+                    tabIndex={-1}
+                  >
+                    {showPassword.password ? (
+                      <svg className="w-4 h-4" fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Biome (2.4.7)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)